src/lib.rs

```diff
@@ -375,7 +404,9 @@ fn process_string(
             if all_keys.contains(stripped_key) {
                 return Ok(None);
             }
-            let contents = std::fs::read_to_string(&value).map_err(|e| {
+            let mut target_file = relative_dir.clone();
+            target_file.push(&value);
```

Comment:
As per the doc recommendation, we should use `Path::join` rather than clone+push: https://doc.rust-lang.org/std/path/struct.Path.html#method.join

Reply:
Perfect.